ONSdigital/census-rm-deploy
# Update groups and names

# Motivation and Context
Make jobs names and groups clearer and more useable. Making the `Overview` the first group on the list means it is the default screen when you click the pipeline on the UI, making it easier to view the whole pipeline.

# What has changed
* Update job names to be shorter and clearer
* Update groups to put and overview first

# How to test?
View the dummy versions I have flown to the sandbox concourse to see the new job names and groups in place 
https://concourse.rm.census-gcp.onsdigital.uk/teams/main/pipelines/adam-test-ci-wl-no-run
https://concourse.rm.census-gcp.onsdigital.uk/teams/main/pipelines/adam-test-performance-no-run

# Links
https://trello.com/c/VxeoYNmw/418-make-concourse-groups-and-jobs-clearer#
